jetty-start/src/main/java/org/eclipse/jetty/start/StartArgs.java

```diff
@@ -208,6 +208,7 @@ public class StartArgs
     private boolean listConfig = false;
     private boolean version = false;
     private boolean dryRun = false;
+    private boolean multiLine = false;
```

Comment:
This field is never read.

Reply:
It is passed in the constructor of CommandLineBuilder at line 523